### ISSUE TITLE
Unify Event/EventRecord for usability

### ISSUE THREAD

**Issue by bobthemighty** [madedotcom/photon-pump#42]

**Comment by bobthemighty** [comment]
The differences between Event, EventRecord, original_event, linked_event etc. are confusing for new users.
 
There's no reason why an Event object can't proxy the underlying EventRecord so that I can simply write

```
async def get_an_event(c):
    e = c.get_event('stream', 1)
    print(e.json())
```

and have that work.

**Comment by bobthemighty** [comment]
Fixed by #59
